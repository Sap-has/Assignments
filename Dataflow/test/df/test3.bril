@main {
  a: int = const 1;
  b: int = const 2;
  c: int = add a b;
  a: int = const 5;
  d: int = add a b;
  print d;
}