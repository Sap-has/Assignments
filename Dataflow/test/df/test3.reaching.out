b1:
  in:  âˆ…
  out: a.b1.3, b.b1.1, c.b1.2, d.b1.4
